snuba/datasets/errors_replacer.py

```diff
@@ -393,20 +393,40 @@ def process_tombstone_events(
         # not.
         return None
 
-    # XXX: We need to construct a query that works on both event_id columns,
-    # either represented as UUID or as hyphenless FixedString. That's why we
-    # use replaceAll(toString()).
-    where = """\
-        PREWHERE replaceAll(toString(event_id), '-', '') IN (%(event_ids)s)
+    def get_timestamp_condition(msg_field: str, operator: str) -> str:
+        msg_value = message.get(msg_field)
+        if not msg_value:
+            return ""
+
+        timestamp = datetime.strptime(msg_value, settings.PAYLOAD_DATETIME_FORMAT)
+        return f"AND timestamp {operator} toDateTime('{timestamp.strftime(DATETIME_FORMAT)}')"
+
+    from_condition = get_timestamp_condition("from_timestamp", ">=")
+    to_condition = get_timestamp_condition("to_timestamp", "<")
```

Comment:
It'd be more ergonomic for Sentry to have this be an inclusive range, as then we can pass in the max timestamp we have fetched from nodestore.